src/pfe/file-watcher/server/test/functional-test/configs/pfe.config.ts

```diff
@@ -10,13 +10,16 @@
  *******************************************************************************/
 
 export const pfe = {
-    "PROTOCOL": process.env.PFE_PROTOCOL || "http",
+    "PROTOCOL": process.env.PFE_PROTOCOL || process.env.IN_K8 ? "https" : "http",
     "HOST": process.env.PFE_HOST || "localhost",
-    "PORT": process.env.PFE_PORT || "9090",
+    "PORT": process.env.PFE_PORT || process.env.IN_K8 ? "9191" : "9090",
 };
```

Comment:
You can add `BASE_URL` here to concatenate PROTOCOL, HOST and PORT, then all test cases can use `BASE_URL` and don't need to concatenate by itself.

Reply:
Sure, I think we can do that. Create `BASE_URL` and use that for the tests.